test/conversions.jl

```diff
@@ -28,3 +28,27 @@ zdt_warsaw = ZonedDateTime(dt, warsaw; from_utc=true)
 # Identical since ZonedDateTime is immutable
 @test astimezone(zdt_utc, warsaw) === zdt_warsaw
 @test astimezone(zdt_warsaw, utc) === zdt_utc
+
+# ZonedDateTime to Unix timestamp (and vice versa)
+@test TimeZones.zdt2unix(ZonedDateTime(1970, utc)) == 0
+@test TimeZones.unix2zdt(0) == ZonedDateTime(1970, utc)
+
+no_dst = DateTime(2013, 2, 13)
+no_dst_zdt = ZonedDateTime(no_dst, warsaw)
+no_dst_offset = 3600
+@test TimeZones.zdt2unix(no_dst_zdt) == datetime2unix(no_dst) - no_dst_offset
```

Comment:
What about this? You can drop the `for` loop if you prefer. The advantage of this approach is that we avoid hardcoding the offset.
```julia
warsaw = TimeZone("Europe/Warsaw")
for dt in (DateTime(2013, 2, 13), DateTime(2016, 8, 11))
	zdt = ZonedDateTime(dt, warsaw)
	offset = TimeZones.value(zdt.zone.offset)  # Total offset in seconds
	@test TimeZones.zdt2unix(zdt) == datetime2unix(dt) - offset
end
```